src/git/zcl_abapgit_git_porcelain.clas.abap

```diff
@@ -4,31 +4,32 @@ CLASS zcl_abapgit_git_porcelain DEFINITION
   CREATE PUBLIC .
 
   PUBLIC SECTION.
-
-    TYPES:
-      BEGIN OF ty_pull_result,
-        files   TYPE zif_abapgit_definitions=>ty_files_tt,
-        objects TYPE zif_abapgit_definitions=>ty_objects_tt,
-        branch  TYPE zif_abapgit_definitions=>ty_sha1,
-      END OF ty_pull_result .
-    TYPES:
-      BEGIN OF ty_push_result,
-        new_files     TYPE zif_abapgit_definitions=>ty_files_tt,
-        branch        TYPE zif_abapgit_definitions=>ty_sha1,
-        updated_files TYPE zif_abapgit_definitions=>ty_file_signatures_tt,
-        new_objects   TYPE zif_abapgit_definitions=>ty_objects_tt,
-      END OF ty_push_result .
+    TYPES: BEGIN OF ty_pull_result,
+             files   TYPE zif_abapgit_definitions=>ty_files_tt,
+             objects TYPE zif_abapgit_definitions=>ty_objects_tt,
+             branch  TYPE zif_abapgit_definitions=>ty_sha1,
+             commit  TYPE zif_abapgit_definitions=>ty_sha1,
+           END OF ty_pull_result .
+    TYPES: BEGIN OF ty_push_result,
+             new_files     TYPE zif_abapgit_definitions=>ty_files_tt,
+             branch        TYPE zif_abapgit_definitions=>ty_sha1,
+             updated_files TYPE zif_abapgit_definitions=>ty_file_signatures_tt,
+             new_objects   TYPE zif_abapgit_definitions=>ty_objects_tt,
+           END OF ty_push_result .
```

Comment:
editing the class via SE24 will format this back to what it was before, makes it more difficult to review

Reply:
Yes, will change it.